index.js

```diff
@@ -214,7 +264,7 @@ const start = async ([exportPath, outputPath = `${process.cwd()}/export`]) => {
         .filter(
           session =>
             !(
-              ((type === 'ride' || type === 'run') &&
+              ((type === 'cycling' || type === 'run') &&
```

Comment:
Not sure if I need to adjust here anything else?

Reply:
Can we keep name it "ride"? I think most folks are moving to Strava and I use this to name the activitity title here: https://github.com/glennreyes/runtastic-gpx/blob/master/index.js#L166-L170.  I know this part can be improved a lot, eg. ideally all cycling related activities are going to be titled into "Ride".

Reply:
Renamed it

Reply:
Alternatively we can keep it cycling, but improve the naming I mentioned here: https://github.com/glennreyes/runtastic-gpx/blob/master/index.js#L166-L170

Reply:
Opened https://github.com/glennreyes/runtastic-gpx/issues/6 for this 👍